x-pack/plugins/security_solution/public/common/containers/source/index.tsx

```diff
@@ -187,7 +192,7 @@ export const useFetchIndex = (
       abortCtrl.current.abort();
       asyncSearch();
     },
-    [data.search, addError, addWarning, onlyCheckIfIndicesExist]
+    [data.search, addError, addWarning, onlyCheckIfIndicesExist, setLoading, setState]
```

Comment:
I thought that we did not have to add React.dispatch from `useState` as dependency